### ISSUE TITLE
Build failure, seemingly wiht easytest 0.3

### ISSUE THREAD

**Issue by snoyberg** [simonmichael/hledger#983]
```
[12 of 49] Compiling Hledger.Utils.Test ( Hledger/Utils/Test.hs, dist/build/Hledger/Utils/Test.o )

Hledger/Utils/Test.hs:101:57: error:
    Not in scope: ‘E.ok’
    Module ‘EasyTest’ does not export ‘ok’.
    |
101 | expectEqPP expected actual = if expected == actual then E.ok else E.crash $
    |                                                         ^^^^

Hledger/Utils/Test.hs:114:9: error:
    Not in scope: ‘E.io’
    Module ‘EasyTest’ does not export ‘io’.
    |
114 |   ep <- E.io (runParserT (evalStateT (parser <* eof) mempty) "" input)
    |         ^^^^

Hledger/Utils/Test.hs:127:10: error:
    Not in scope: ‘E.io’
    Module ‘EasyTest’ does not export ‘io’.
    |
127 |   eep <- E.io $ runExceptT $
    |          ^^^^

Hledger/Utils/Test.hs:143:9: error:
    Not in scope: ‘E.io’
    Module ‘EasyTest’ does not export ‘io’.
    |
143 |   ep <- E.io (runParserT (evalStateT parser mempty) "" input)
    |         ^^^^

Hledger/Utils/Test.hs:160:10: error:
    Not in scope: ‘E.io’
    Module ‘EasyTest’ does not export ‘io’.
    |
160 |   eep <- E.io $ runExceptT $ runParserT (evalStateT parser mempty) filepath input
    |          ^^^^
```

**Comment by simonmichael** [comment]
Thanks for the report. 
Related: https://github.com/joelburget/easytest/issues/22

**Comment by joelburget** [comment]
Whoops, yeah we need to either put an upper bound on easytest or convert to 0.3. I can open a PR with my conversion if you want.

**Comment by simonmichael** [comment]
Yes please!

**Comment by joelburget** [comment]
@simonmichael I just opened the PR. It would probably also be a good idea to add a hackage revision upper-bounding easytest to (`< 0.3`) for `hledger` 1.11 - 1.14.1.

**Comment by simonmichael** [comment]
Quite right, I'll do that.

**Comment by simonmichael** [comment]
This was resolved by adding bounds, easytest 0.3 discussion continues on https://github.com/simonmichael/hledger/pull/987.
